app/Belinska/conftest.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+
+from app.task import Task
+
+from app.to_do_list import ToDoList
+
+
+@pytest.fixture
+def general_task(task_data):
```

Comment:
brackets of a fixture function is a place for other fixture. It means we can pass a fixture to an other fixture. Here you are trying to pass the `task_data` fixture.  That is why the tests that use this fixture fail.